src/content/plugins/chunks-webpack-plugin.md

```diff
@@ -0,0 +1,322 @@
+---
+title: ChunksWebpackPlugin
+contributors:
+  - yoriiis
+related:
+  - title: SplitChunksPlugin
+    url: /plugins/split-chunks-plugin
+  - title: The real power of Webpack 4 SplitChunks Plugin
+    url: https://medium.com/@Yoriiis/the-real-power-of-webpack-4-splitchunks-plugin-fad097c45ba0
+---
+
+The ChunksWebpackPlugin creates HTML files with entry points and chunks relations to serve your webpack bundles. It is suitable with multiple page application which contains multiple entry points.
+
+Since Webpack 4, SplitChunksPlugin offer the possibility to optimizes all chunks. It can be particularly powerful, because it means that chunks can be shared even between async and non-async chunks. See the webpack documentation for [`splitChunks.chunks`](https://webpack.js.org/plugins/split-chunks-plugin/#splitchunkschunks).
```

Comment:
```suggestion
Since webpack 4, `SplitChunksPlugin` offers the possibility to optimizes all chunks. It can be particularly powerful, because it means that chunks can be shared even between async and non-async chunks. See the webpack documentation for [`splitChunks.chunks`](/plugins/split-chunks-plugin/#splitchunkschunks) for details.
```